miso-web/src/main/webapp/scripts/hot_library.js

```diff
@@ -75,6 +79,31 @@ HotTarget.library = (function() {
 					if (match) {
 						setData(match.label);
 					}
+        } else if (flatProperty === 'index' + (n - 1) + 'Label' && !Utils.validation.isEmpty(value)){
+          var pt = getPlatformType(flat.platformType);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "pt" as this name is already used in declaration at line 48. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2814)